src/components/custom/signup.tsx

```diff
@@ -69,9 +71,17 @@ export const Appearances = {
     public: 'public',
     newSignup: 'newSignup',
 }
+const query = graphql`
```

Comment:
@meenakshi-deriv let's use the new `StaticImage` per @yashim-deriv suggestion. Thanks 🙏